inference/server/oasst_inference_server/user_chat_repository.py

```diff
@@ -108,17 +108,15 @@ def initiate_assistant_message(self, parent_id: str, work_parameters: inference.
         self.session.refresh(message)
         return message
 
-    def add_vote(self, message_id: str, score: int) -> models.DbVote:
-        logger.info(f"Adding vote to {message_id=}: {score=}")
+    def update_score(self, message_id: str, score: int):
+        if score < -1 and score > 1:
+            raise fastapi.HTTPException(status_code=400, detail="Invalid score")
```

Comment:
Is this the correct way to validate input in fastapi? Please correct me if I'm wrong

Reply:
I think technically we could use [pydantic's built-in constraints](https://docs.pydantic.dev/usage/types/#arguments-to-conint) to constrain the `score` field of `VoteRequest` but this use case is really simple so I don't think this is a problem. Although I think you need to change `and` to `or`

Reply:
ah thanks, it was a quick code.